test/e2e/query_test.go

```diff
@@ -368,6 +368,21 @@ func TestQueryWithAuthorizedSidecar(t *testing.T) {
 	})
 }
 
+func TestQueryActivePath(t *testing.T) {
+	t.Parallel()
+
+	e, err := e2e.NewDockerEnvironment("e2e_test_query_active")
+	testutil.Ok(t, err)
+	t.Cleanup(e2ethanos.CleanScenario(t, e))
+
+	qBuilder := e2ethanos.NewQuerierBuilder(e, "1").WithActivePath("test-active")
+	q := qBuilder.Init()
+	testutil.Ok(t, e2e.StartAndWaitReady(q))
+
+	_, err = os.Stat(filepath.Join(q.InternalDir(), "test-active", "queries.active"))
+	testutil.Assert(t, os.IsExist(err)) // fails if os.IsExist() is false.
```

Comment:
So I think this might fail, as you aren't executing a query against this Querier and it won't have any active queries file.